docs/index.rst

```diff
@@ -14,28 +13,30 @@ bioframe
    :maxdepth: 2
    :caption: Guide
 
-   quickstart
-   notebooks/intervals_tutorial.ipynb
-   notebooks/performance.ipynb
-   notebooks/recipes.ipynb
-   technical-notes
+   guide-quickstart
+   guide-intervalops.md
```

Comment:
it might be better if the subsections didn't show up on the overall toctree 
![image](https://user-images.githubusercontent.com/19657573/121957461-47f34800-cd17-11eb-919a-7929e786702c.png)